helpers/auth_provider.py

```diff
@@ -2,26 +2,29 @@
 in the application to authenticate users.
 """
 
+from typing import Dict
+from typing import Union
+
 from flask_httpauth import HTTPBasicAuth
 from werkzeug.security import check_password_hash
 
 
 class HTTPAuthProvider(HTTPBasicAuth):
     """User authentication provider class(Singleton)."""
-    def __init__(self):
+    def __init__(self) -> None:
         HTTPBasicAuth.__init__(self)
         self.verify_password(self._verify_password)
-        self.credentials = {}
+        self.credentials: Dict[str, str] = {}
 
-    def _verify_password(self, username, password):
+    def _verify_password(self, username: str, password: str) -> Union[str, bool]:
         """Verify credentials."""
-        if username in self.credentials and \
-                check_password_hash(self.credentials.get(username), password):
+        pwhash = self.credentials.get(username)
```

Comment:
Not related with the commit.
What's the code style rule being violated here?

Reply:
rolled back